share/completions/pkg.fish

```diff
@@ -165,3 +165,105 @@ complete -c pkg -n '__fish_pkg_is list' -xa '(pkg query "%n")'
 
 # update
 complete -c pkg -n '__fish_pkg_is add update' -s f -l force -d "Force a full download of a repository"
+
+# alias 
+for alias in (pkg alias -q)
```

Comment:
This is executed when the completion file is loaded. That means it won't pick up new aliases (or throw away old ones).

If you wanted to make it dynamic, you'd have to put this into a function, print the things as "name\tdescription" (that's a tab character) and call it in `complete -a`, like

```fish
complete -c pkg -n __fish_pkg_subcommand -xa '(__fish_pkg_aliases)'
```

Of course this depends on if `pkg alias -q` is usably fast. If not, having these cached isn't too bad.